src/borg/archiver.py

```diff
@@ -3823,6 +3851,21 @@ def define_borg_mount(parser):
 
         See the "borg help patterns" command for more help on exclude patterns.
 
+        Option ``--bypass-lock`` allows you to access the repository while bypassing
+        borg's locking mechanism. This is necessary if your repository is on a read-only
+        storage where you don't have write permissions or capabilities and therefore
+        cannot create a lock. Examples are repositories stored on a Bluray disc or a
+        read-only network storage. Avoid this option if you are able to use locks as
+        that is the safer way; see the warning below.
+
+        .. warning::
+
+            If you do use ``--bypass-lock``, you are responsible to ensure that no other
+            borg instances have write access to the repository. Otherwise, you might
+            experience errors and read broken data if changes to that repository are
+            being made at the same time.
+
+
```

Comment:
I guess we do not want to copy that text to all borg subcommands that should support --bypass-lock.
Guess we should make it a global options even if not all subcommands will support it.

Reply:
Do you mean you don't want to insert the text itself multiple times into the code? (We could define `BYPASS_LOCK_HELP_TEXT` somewhere and add that to the epilog/help of each command.) Or do you not want to display the text for each command as it would clutter the help page with that big warning which only concers a few people?

Reply:
well, primarily i do not want n-fold duplication within the source.

but also, if something is needed frequently, that's also a reason for a global option, even if we can't/won't support it for all commands.

Reply:
needed frequently -> supported by a lot of commands

Reply:
Yup, agreed. I'll take a look how to move it to the common options.